pages/content/posts/complete-interop-features/index.md

```diff
@@ -0,0 +1,73 @@
+
+Since the early 2010s (and even prior), standard bodies and browser makers have rapidly specified and implemented new APIs. MDN's goal was to document these new features. As these features were web developers' main focus, extensive, accurate, and current documentation was (and continues to be) required. It was an easier move to sell them internally as new technologies led to more engagement. Getting resources to document these APIs was more straightforward than the old HTML table features. The originally named HTML5, CSS3, and ES6 initiatives allowed MDN to revamp these areas of documentation because there was an interest in positioning the web as a recognized interoperable platform.
+
+
+![The three logos of HTML5, CSS3, and JS](logos.png)
+
+The combination of scarcity of resources and the high-speed development of the web platform led to the original gaps staying open. Some mitigating actions were taken, like listing every method and property for each interface. However, there was still a large gap of more than 1000 missing pages.
+
+Today's gaps mostly fall into three categories:
+
+- HTML DOM, and the SVG DOM features added to the web platform when MDN was still in its infancy;
+- non-interoperable features: APIs supported by only one or two browser engines and missed by the browser makers' writers;
+- single properties and methods that were forgotten when their API was initially documented or added later (but they slipped between the cracks of the different writing processes).
+
+## Opportunities created by the Interop initiative
+
+As a collective of writers dedicated to helping document the open web platform, [Open Web Docs (OWD)](https://openwebdocs.org/) is helping _close these gaps_. Unlike writers who are paid by browser makers needing to justify their work, OWD's backers, individuals and companies, gave us the more generic mission to improve open web documentation. We are able to focus on much-needed activities that may not bring the most engagement at first, but significantly improve in the middle- and long-term situations. We are able to focus on these valuable activities that benefit developers and the longevity of the documentation, without having anybody's internal projects dictate our priorities.
```

Comment:
```suggestion
As a collective of writers dedicated to documenting the open web platform, [Open Web Docs (OWD)](https://openwebdocs.org/) is able to _close these gaps_.  We believe that projects like this can significantly improve in the middle- and long-term health and the longevity of the documentation. We aim to provide complete documentation of the web platform.
```